config/constants.ts

```diff
@@ -291,7 +291,7 @@ External resources required by the wgtsqc Stack
 // Deployed under dev/stg/prod
 export const wgtsQcIcav2PipelineIdSSMParameterPath = '/icav2/umccr-prod/wgts_qc_4.2.4_pipeline_id'; // 03689516-b7f8-4dca-bba9-8405b85fae45
 
-export const wgtsQcIcav2PipelineWorkflowType = 'wgtsQc';
+export const wgtsQcIcav2PipelineWorkflowType = 'wgts-qc';
 export const wgtsQcIcav2PipelineWorkflowTypeVersion = '4.2.4';
```

Comment:
Ah let me fix bsshFastqCopy too, bsshFastqCopy is not used in any output paths so got less attention but I think it would be good to fix for workflow run name consistency

Reply:
btw: are we OK with `cttsov2`?

First, we are adding a version where the workflow definition has already a `version` field. That feels redundant and confusing and may limit our future uses of that workflow (e.g. what if there's a new version 3.0.1?)
Second, we've seen now multiple other places where the analysis and the sample type are separated, e.g. tso-ctdna. I wonder if we are limiting the possible use cases combining those into one term, e.g. could there be a tso-ctrna, a tso600-ctdna or others? 

I just want to take this opportunity to rethink our workflow naming based on the experience we have now, not simply following the convenience of what we used so far. 
I am not the best person to judge this though, so I'll follow the consensus

Reply:
Maybe a discussion on Slack where we can reach a consensus.